verl/experimental/agent_loop/agent_loop.py

```diff
@@ -275,14 +276,24 @@ def get_agent_loop_class(self, agent_name: str) -> Type[AgentLoopBase]:
         Raises:
             ValueError: If the agent_name is not recognized.
         """
-        # TODO: add tool agent registrary
         from verl.experimental.agent_loop.single_turn_agent_loop import SingleTurnAgentLoop
         from verl.experimental.agent_loop.tool_agent_loop import ToolAgentLoop
 
         if agent_name == "single_turn_agent":
             return SingleTurnAgentLoop
         elif agent_name == "tool_agent":
             return ToolAgentLoop
+
+        cfg_agent = getattr(
+            self.config.actor_rollout_ref.rollout.agent,
```

Comment:
cfg_agent = self.config.actor_rollout_ref.rollout.agent.get("custom_agent_loop") should just work?

Reply:
could u add an unit test for the custom agent? thanks.
if you come across legacy config test error, you can ignore it. will be fixed by https://github.com/volcengine/verl/pull/2433

Reply:
@eric-haibin-lin I'm pending a successful test run on the new unit test that I added, but is this an acceptable test in general? I just wanted to add a dummy test to simplify it as much as possible.